js/map-content-to-topper.js

```diff
@@ -0,0 +1,162 @@
+const themeImageRatio = {
+	'split-text-center': 'split',
+	'split-text-left': 'split',
+	'full-bleed-image-center': 'full-bleed',
+	'full-bleed-image-left': 'full-bleed',
+	'full-bleed-offset': 'full-bleed'
+};
+
+const isNews = content => content.annotations && content.annotations.find(a => a.prefLabel === 'News');
+
+const getTopperSettings = content => {
+	content.topper = content.topper || {};
+	//live blogs and news packages
+	if (content.realtime && content.liveBlog ||
+		content.package && isNews(content.package) && content.package.contains[0].id === content.id ||
```

Comment:
`content.package` isn't a thing in ElasticSearch. it's something that's grafted onto the content from a big ol' next-api query.